src/Symfony/Component/Console/Formatter/OutputFormatter.php

```diff
@@ -33,6 +33,18 @@ public static function escape($text)
     {
         $text = preg_replace('/([^\\\\]?)</', '$1\\<', $text);
 
+        return self::escapeTrailingBackslash($text);
+    }
+
+    /**
+     * Escapes trailing "\" in given text.
+     *
+     * @param string $text Text to escape
+     *
+     * @return string Escaped text
+     */
+    public static function escapeTrailingBackslash($text)
```

Comment:
Not sure. I guess it can be used by anyone creating it's own helpers in userland.